public/app/percona/dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -66,6 +68,18 @@ export class PSMDBService extends DBClusterService {
     );
   }
 
+  getExpectedResources(dbCluster: DBCluster): Promise<DBClusterExpectedResources> {
+    return apiManagement
+      .post<any, Partial<DBClusterPayload>>('/DBaaS/PSMDBCluster/Resources/Get', pick(toAPI(dbCluster), ['params']))
+      .then((response: DBClusterExpectedResourcesAPI) => ({
+        expected: {
+          cpu: response.expected.cpu_m / 1000,
+          memory: response.expected.memory_bytes / 10 ** 9,
+          disk: response.expected.disk_size / 10 ** 9,
```

Comment:
I'd use constants instead of those hardcoded values

Reply:
Done @nicolalamacchia, thanks!